src/main/java/org/broadinstitute/dsm/DSMServer.java

```diff
@@ -252,7 +269,7 @@ protected void setupCustomRouting(@NonNull Config cfg) {
     protected void updateDB(@NonNull String dbUrl) {
         logger.info("Running DB update...");
 
-        try (Connection conn = DriverManager.getConnection(dbUrl + "&verifyServerCertificate=true&useSSL=true&requireSSL=true"
+        try (Connection conn = DriverManager.getConnection(dbUrl
```

Comment:
Did you update the JDBC URL to use the GCP options?

Reply:
Yep!